packages/x-charts/src/hooks/useSeries.ts

```diff
@@ -1,16 +1,16 @@
 import * as React from 'react';
-import { SeriesContext } from '../context/SeriesContextProvider';
+import { FormattedSeries, SeriesContext } from '../context/SeriesContextProvider';
 
 /**
  * Get access to the internal state of series.
  * Structured by type of series:
  * { seriesType?: { series: { id1: precessedValue, ... }, seriesOrder: [id1, ...] } }
  * @returns FormattedSeries series
  */
-export function useSeries() {
-  const series = React.useContext(SeriesContext);
+export function useSeries(): FormattedSeries {
+  const { isInitialized, ...series } = React.useContext(SeriesContext);
```

Comment:
I don't see reason of not doing this split between the `data` and the `isInitialized`. Since only hooks are user-facing, we can add the `data` in context and return the `data` when using the hook.

Reply:
I had it ready, I just spent a lot of time trying to understand why the karma tests are failing when using the `ErrorBoundary`. It seems it is an issue with our karma setup here: https://github.com/mui/material-ui/blob/next/packages-internal/test-utils/src/setupKarma.js
and also possibly that we override the `console` somewhere I couldn't find. But in the end I couldn't find a proper solution to it, so I just skip it in karma 😓

Reply:
Thanks for the input btw